ubuntudesign/developer.ubuntu.com
# Update to new ./run

Update to new ./run command.

## QA

0. Travis should pass and lint, excluding flake8 error I am just about to fix in a separate PR
1. `./run` should work correctly (you may need to clean up old files from old ./run)
2. http://localhost:8015/core should load nicely
